test/metabase/api/action_test.clj

```diff
@@ -305,3 +305,58 @@
           (testing "Should not have committed any of the valid rows"
             (is (= 75
                    (categories-row-count)))))))))
+
+(deftest bulk-delete-happy-path-test
+  (testing "POST /api/action/bulk/delete/:table-id"
+    (mt/test-drivers (mt/normal-drivers-with-feature :actions)
+      (actions.test-util/with-actions-test-data-and-actions-enabled
+        (is (= 75
+               (categories-row-count)))
+        (is (= {:created-rows [{(format-field-name :id) 76, (format-field-name :name) "NEW_A"}
+                               {(format-field-name :id) 77, (format-field-name :name) "NEW_B"}]}
+               (mt/user-http-request :crowberto :post 200
+                                     (format "action/bulk/create/%d" (mt/id :categories))
+                                     [{(format-field-name :name) "NEW_A"}
+                                      {(format-field-name :name) "NEW_B"}])))
```

Comment:
You don't need to be creating new rows to test `delete`. Even if you did, I don't think you should use the `bulk/create` action to do it. This test is supposed to test `bulk/delete`, so it would be nice to test that independently of everything else if possible. Otherwise if I accidentally break `bulk/create` then both the `bulk/create` and `bulk/delete` tests would start failing. It's going to make fixing things harder since I might not be sure immediately that I only broke `bulk/create` and the `bulk/delete` failure is a red herring 🐟 as opposed to some larger bug that broke both e.g. a bug in the nested transaction handling code.

 `with-actions-test-data-and-actions-enabled` creates a new Database every time and trashes it at the end. So you can delete whatever you want inside of it and it won't affect anyone else.